demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>There is no "deep machine learning" specialization. You might want to say, "The StellarGraph library supports many state-of-the-art graph machine learning algorithms..." You might also want to put a link to our introductory Medium article that already goes into detail about GCN (https://medium.com/stellargraph/knowing-your-neighbours-machine-learning-on-graphs-9b7c3d0d5896).</p><p> </p><p>I think "sometimes called node classification or even node attribute inference." can be safely removed from this sentence without loss or simplify to say " commonly known as node classification." As long as you consistently use the latter term in the rest of the document and not use the "node attribute inference" term then you don't have to mention it here.</p><p> </p><p>You might want to mention that Cora is a directed graph but we are going to treat it as undirected.</p><p> </p><p> </p><blockquote>StellarGraph supports <a href="https://github.com/stellargraph/stellargraph/raw/d09fefd5325ee9f895640defac4006ecac8f1c0b/demos/node-classification/README.md" target="_blank">other algorithms for doing node classification</a>, as well as many <a href="https://github.com/stellargraph/stellargraph/raw/d09fefd5325ee9f895640defac4006ecac8f1c0b/demos/README.md" target="_blank">other tasks</a>. </blockquote><p> </p><p>Should probably drop the word "doing" from above. Also, can we be more specific about the other tasks, and maybe re-write to say "as well as, link prediction, interpretabililty, community detection, graph classification, etc." and link each to the individual folder?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Yes, that should be fine.</p>